10_kata.py

```diff
@@ -0,0 +1,8 @@
+# https://www.codewars.com/kata/5287e858c6b5a9678200083c/train/python
+
+def narcissistic(value):
+
+    digits = list(str(value))
+    number_of_digits = len(str(value))
+    sum_of_number = sum(int(x) ** number_of_digits for x in digits)
```

Comment:
Here it could be useful to make the `x` variable a little bit more descriptive. For example you could rename it to `digit`, then it's more clear that in a loop, every iteration uses a single `digit` from the `digits` variable. 

Although I know it's a small thing, and in this simple code it's pretty clear what the `x` is, but that is a good habit to have and in a bigger piece of code, especially when someone else is looking at it, it's good to keep things as descriptive as possible.